nengo_loihi/tests/test_simulator.py

```diff
@@ -0,0 +1,15 @@
+import nengo
+import pytest
+
+import nengo_loihi
+
+
+def test_cx_model_validate_notempty(Simulator):
+    with nengo.Network() as model:
+        nengo_loihi.add_params(model)
+
+        a = nengo.Ensemble(10, 1)
+        model.config[a].on_chip = False
+
+    with pytest.raises(RuntimeError):
+        Simulator(model)
```

Comment:
I'm not sure if this makes a difference (since I believe the error happens in `__init__`) but in similar tests I've been creating the simulator as a context manager here with the thought that the `__exit__` might still fire and clean things up if anything was made.

Reply:
Sure, I'm fine with that.